cPP_DSC.adoc

```diff
@@ -1177,7 +1177,7 @@ FCS_COP.1.1/SKC:: The TSF shall perform [_symmetric-key encryption/decryption_]
 
 FCS_RBG.1 Random Bit Generation (RBG)
 
-FCS_RBG.1.1:: The TSF shall perform deterministic random bit generation services using [*selection*: _RBG algorithm_] in accordance with [*selection*: _list of standards_] after initialization with a seed.
+FCS_RBG.1.1:: The TSF shall perform deterministic random bit generation services using [*selection*: _DRBG algorithm_] in accordance with [*selection*: _list of standards_] after initialization *with a seed*.
```

Comment:
the addition of "with a seed" is a refinement on the CC22 SFR?

Reply:
Yes it is